OEBPS/Chapter4.xhtml

```diff
@@ -56,7 +56,7 @@
             <p>Harry scowled. "Draco was only hit with his own stunner, but Professor Snape had him brought to the infirmary instead of reviving him immediately. I believe he did that so they could work out a story to tell. From what I've read there is a spell that will show the last spells cast by a wand. Would you mind terribly casting it on my wand so you can be a witness that the only thing I did was cast a shield charm?"</p>
             <p>She looked at him a little oddly, but nodded. "Very well, hand me your wand, Mr. Potter."</p>
             <p>Harry did so, though he now felt a little exposed without it.</p>
-            <p>"<em>Priori Incantato!</em>" she said as she tapped his wand with her own. An ethereal looking shield emerged from the tip, followed by a transfiguration spell she recognized from yesterday's homework assignment. "That confirms your story, though I feel I should congratulate you. That's fairly advanced defensive magic for someone of your age." She smiled a little as she handed him his wand back.</p>
+            <p>"<em>Prior Incantato!</em>" she said as she tapped his wand with her own. An ethereal looking shield emerged from the tip, followed by a transfiguration spell she recognized from yesterday's homework assignment. "That confirms your story, though I feel I should congratulate you. That's fairly advanced defensive magic for someone of your age." She smiled a little as she handed him his wand back.</p>
```

Comment:
Oh, good catch. I had to google this one to check my memory on which one was which!